config/core/test-configs.yaml

```diff
@@ -357,6 +368,12 @@ test_plans:
   smc:
     rootfs: buildroot_baseline_ramdisk
 
+  tast:
+    rootfs: chrome_version
+    pattern: 'chromeos/new-chromeos-template.jinja2'
+    filters:
+      - passlist: {defconfig: ['x86_64_defconfig+x86-chromebook']}
```

Comment:
This is specific to devices, not to the test plan

Reply:
updated